includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -552,6 +552,20 @@ public function javascript_params() {
 		$stripe_params['isAmazonPayAvailable'] = WC_Stripe_Feature_Flags::is_amazon_pay_available();
 
+		/**
+		 * Filters the list of permitted font domains for the Stripe Payment Element.
+		 * These host names will be used in the client Javascript to identify additional domains
+		 * that return text/css resources which include @font-face declarations. Those stylesheets
+		 * can then be passed into the Stripe payment element.
+		 *
+		 * @param string[] $permitted_font_domains List of permitted font domains.
+		 * @since 10.6.0
+		 */
+		$permitted_font_domains = apply_filters( 'wc_stripe_upe_permitted_font_domains', [] );
```

PHPMD (2.15.0)

[error] The variable $permitted_font_domains is not named in camelCase. (undefined)

(CamelCaseVariableName)